ssds/blobstore/__init__.py

```diff
@@ -33,11 +35,16 @@ def put(self, bucket_name: str, key: str, data: bytes):
     def cloud_native_checksum(self, bucket_name: str, key: str) -> str:
         raise NotImplementedError()
 
+    def parts(self, bucket_name: str, key: str, executor: ThreadPoolExecutor=None) -> "AsyncPartIterator":
```

Comment:
Any reason that these (`"AsyncPartIterator"`) are in quotes?

Reply:
`AsyncPartIterator` is defined later in this file, which confuses type hints. The workaround is to use a string.